packages/web/src/components/shared/RecentSvg.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import types from '@appbaseio/reactivecore/lib/utils/types';
+
+const RecentSvg = ({ className, icon }) => {
```

Comment:
@anik-ghosh-au7 Why are you repeating the `PopularSvg` component?
Both components have the same functionality. The only change is the SVG.
Create a standalone component and change the svg with some prop.